book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] A method for adding rentals should be present then the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
It can be similar to 👇🏾 👇🏾 

```ruby
  def add_rental(person, date) 
    Rental.new(date, self, person)
  end
```